opensquirrel/parser/libqasm/parser.py

```diff
@@ -170,10 +170,11 @@ def _get_gate_generator(self, instruction: cqasm.semantic.GateInstruction) -> Ca
             if gate_name == "inv":
                 return InverseGateModifier(modified_gate_generator)
             if gate_name == "pow":
-                return PowerGateModifier(instruction.gate.parameter.value, modified_gate_generator)
+                return PowerGateModifier(instruction.gate.parameters[0].value, modified_gate_generator)
```

Comment:
```suggestion
                gate = instruction.gate
                exponent = gate.parameters[0]
                return PowerGateModifier(instruction.gate.parameters[0].value, modified_gate_generator)
```

Just a suggestion. It maybe clarifies the code.